caluma/caluma_form/tests/test_jexl.py

```diff
@@ -411,6 +411,50 @@ def test_answer_transform_in_tables(
         validator.validate(document, info)
 
 
+def test_answer_transform_hidden_table_cell(
+    info,
+    form_and_document,
+    form_question_factory,
+):
+    form, document, questions, answers = form_and_document(
+        use_table=True, use_subform=False
+    )
+    column_a = answers["column"]
+    column_a.value = 11.0
+    column_a.save()
+
+    table_question = questions["table"]
+
+    # create column that is always hidden
+    col2_question = form_question_factory(
+        form=table_question.row_form,
+        question__slug="column2",
+        question__is_hidden="true",
+        question__is_required="false",
+    ).question
+
+    # create question that depends on the value of the previously created hidden
+    # column
+    form_question_factory(
+        form=form,
+        question__slug="check",
+        question__is_hidden=f"('yes' in '{table_question.slug}'|answer|mapby('{col2_question.slug}'))",
+        question__is_required="true",
+    ).question
```

Comment:
The `.question` is not required here, we're not using the return value anyways

Reply:
done